library/ui-strings/src/main/res/values/strings.xml

```diff
@@ -442,6 +442,9 @@
     <string name="labs_enable_new_app_layout_title">Enable new layout</string>
     <string name="labs_enable_new_app_layout_summary">A simplified Element with optional tabs</string>
 
+    <string name="labs_enable_deferred_dm_title">Enable deferred DMs</string>
+    <string name="labs_enable_deferred_dm_summary">Direct rooms will be created after sending a first message.</string>
```

Comment:
The wording may be a bit technically confusing, maybe 
```suggestion
    <string name="labs_enable_deferred_dm_summary">Direct rooms will be created when sending a first message.</string>
```
is better. WDYT?
(Not a blocker)

Reply:
I asked @gaelledel on this point, just waiting for her feedback to update the wording